apps/utils.py

```diff
@@ -80,3 +80,15 @@ class Labels:
         PENDING = 'PENDING'
         COMPLETE = 'COMPLETE'
         FAILED = 'FAILED'
+
+
+def get_organization_name(request):
+    if request.user.is_authenticated():
```

Comment:
`is_authenticated` is a property on TokenUser so you don't have to make the function call here